opencog/openpsi/main.scm

```diff
@@ -17,6 +17,8 @@
 "
   psi-rule-nocheck -- same as psi-rule, but no checking
 "
+    (define implication (Implication a-stv (SequentialAnd context action) goal))
```

Comment:
Again I think the `SequentialAnd` is ill-used, better use a `PredictiveImplication`

See http://wiki.opencog.org/wikihome/index.php/PredictiveImplicationLink